onnxruntime/test/shared_lib/test_tensor_loader.cc

```diff
@@ -17,11 +17,11 @@ namespace test {
 
 TEST_F(CApiTest, load_simple_float_tensor_not_enough_space) {
   // construct a tensor proto
-  onnx::TensorProto p;
+  ONNX_NAMESPACE::TensorProto p;
```

Comment:
This file doesn't need be changed

Reply:
Out of curiosity, why we don't want to make the changes for these tests, test_tensor_loader.cc and test_model_loading.cc?

Reply:
Even you don't change them, it shouldn't matter. It won't break your internal build.

Reply:
OK, that's a fair point. I will revert the changes to these two tests and the cmake file.

Reply:
Actually, I tend to keep the change to test_tensor_loader.cc to make it consistent with the rest of the file, where all but this one use ONNX_NAMESPACE. What do you think?

Reply:
Which is the 'rest'?

Reply:
You mean, some file in this dir is using the ONNX_NAMESPACE macro?

Reply:
My bad. I was looking at a previous version of my OWN file. I've reverted the change.